packages/demo-nextjs/README.md

```diff
@@ -0,0 +1,24 @@
+# Next.js App Demo
+
+Typeform can be embedded in server-side rendered Next.js apps.
+
+**Note:** We recommend using `@typeform/embed-react`, it will be rady soon :)
+
+You can use the `@typeform/embed` directly in Next.js apps:
+
+```javascript
+import { createWidget } from "@typeform/embed";
+import "@typeform/embed/build/css/widget.css"; // import necessary CSS
+
+const MyTypeformEmbed = () => {
+  const container = useRef();
+
+  useEffect(() => {
+    createWidget("<form id>", { container: container.current });
```

Comment:
I think I saw somewhere that container was renamed to element. Was it? Not sure though.

Reply:
That PR was not merged. It is container.

Reply:
Ok, then we're cool.